test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -0,0 +1,63 @@
+package seedu.addressbook.common;
+
+import static org.junit.Assert.*;
+
+import java.util.List;
+import java.util.ArrayList;
+
+import org.junit.Test;
+
+public class UtilsTest {
+
+    @Test
+    public void isAnyNull_checkEmptyArgument() {
+        assertFalse(Utils.isAnyNull());
+    }
+    
+    @Test
+    public void isAnyNull_checkSomeNull() {
+        assertTrue(Utils.isAnyNull(null));
+        assertTrue(Utils.isAnyNull("not null", null));
+        assertTrue(Utils.isAnyNull(123, "not null", null));
+    }
+    
+    @Test 
+    public void isAnyNull_allNotNull() {
+        assertFalse(Utils.isAnyNull(1));
+        assertFalse(Utils.isAnyNull(1, "Hello world"));
+        assertFalse(Utils.isAnyNull(false, 1, "Hello world"));
+    }
+    
+    @Test 
+    public void elementsAreUnique_checkEmptyArgument() {
+        assertTrue(Utils.elementsAreUnique(new ArrayList<String>()));
+    }
+    
+    @Test
+    public void elementsAreUnique_uniqueElements() {
+        List<Integer> list;
+        
+        list = makeList(new int[]{1, 2, 3, -1, 0, 500}); 
+        assertTrue(Utils.elementsAreUnique(list));
+        list = makeList(new int[]{0, 400, 123, -100, 1, 12}); 
+        assertTrue(Utils.elementsAreUnique(list));
+    }
+    
+    @Test
+    public void elementsAreUnique_duplicateElements() {
+        List<Integer> list;
+        
+        list = makeList(new int[]{1, 1}); 
+        assertFalse(Utils.elementsAreUnique(list));
+        list = makeList(new int[]{0, 1, 2, 3, 4, 0}); 
```

Comment:
You can use `List<Integer> list = Arrays.asList(1, 1, 1, 2)` to create and initialize a list in one line.